lib/coinatmradar/coinatmradar.js

```diff
@@ -1,11 +1,13 @@
 const axios = require('axios')
 const _ = require('lodash/fp')
 const hkdf = require('futoin-hkdf')
+const bip39 = require('bip39')
```

Comment:
Not necessary. Editor configured for Standard should warn you about this.

```suggestion
```

Reply:
Hum, looks like I'm having some issues with workspace configuration on vscode =p